src/lib/get-osm-stream.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+const url = require('url')
+const request = require('request')
+const Readable = require('stream').Readable
+const { osmapi } = require('../config')
+
+// Query the osm api using a bounding box string, returns a readable stream
+module.exports = function getOsmStream (bbox, cb) {
+  const queryUrl = url.resolve(osmapi, `map?bbox=${bbox}`)
+  request(queryUrl, function (err, response, body) {
```

Comment:
Could we just use [`http.get()`](https://nodejs.org/api/http.html#http_http_get_options_callback)? You can look at `res.statusCode` but still treat `res` as a readable stream.